manifests/l/lin-ycv/EverythingCmdPal/0.1.0/lin-ycv.EverythingCmdPal.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: lin-ycv.EverythingCmdPal
+PackageVersion: 0.1.0.0
+PackageLocale: en-US
+Publisher: lin-ycv

```

```suggestion
Publisher: VictorLin
```

I believe there is a mismatch between the package identity and the WinGet package id here. If you want to keep the WinGet ID as "lin-ycv.EverythingCmdPal" that should do the trick.

---

Thanks for the suggestion!